src/pages/Authentication/ForgetPassword.js

```diff
@@ -54,7 +54,7 @@ const ForgetPasswordPage = (props) => {
     forgetSuccessMsg: state.ForgetPassword.forgetSuccessMsg,
   }));
 
-  document.title = "Reset Password | Velzon - React Admin & Dashboard Template";
+  document.title = "Reset Password |GreenMe";
```

Comment:
Kindly maintain the space after the pipe `|` so it's consistent.

```jsx
document.title = "Reset Password | GreenMe";
```

Reply:
fix it